src/main/kotlin/uk/gov/dluhc/printapi/service/PrintRequestsService.kt

```diff
@@ -7,17 +7,47 @@ import uk.gov.dluhc.printapi.database.entity.Status
 import uk.gov.dluhc.printapi.database.repository.PrintDetailsRepository
 import uk.gov.dluhc.printapi.messaging.MessageQueue
 import uk.gov.dluhc.printapi.messaging.models.ProcessPrintRequestBatchMessage
+import uk.gov.dluhc.printapi.rds.entity.Certificate
+import uk.gov.dluhc.printapi.rds.repository.CertificateRepository
 
 private val logger = KotlinLogging.logger { }
 
 @Component
 class PrintRequestsService(
     private val printDetailsRepository: PrintDetailsRepository,
+    private val certificateRepository: CertificateRepository,
     private val idFactory: IdFactory,
     private val processPrintRequestQueue: MessageQueue<ProcessPrintRequestBatchMessage>,
 ) {
 
     fun processPrintRequests(batchSize: Int) {
+        processDynamoBatch(batchSize)
+        processBatch(batchSize)
+    }
+
+    private fun processBatch(batchSize: Int) {
+        batchCertificates(batchSize).forEach { (batchId, batchOfCertificates) ->
```

Comment:
I have 2 questions here, though I know they are contradictory to each other! 🤣 

1. Should processBatch and processDynamoBatch work from the same call to batchCertificates? 
At the moment they both call it independently , so possibility for there to be 2 different result sets?
2. Should processBatch (which is the one that uses JPA repositories) call a JPA version of batchCertificates?